static/app/components/events/breadcrumbs/breadcrumbsDataSection.tsx

```diff
@@ -146,35 +139,34 @@ export default function BreadcrumbsDataSection({event}: BreadcrumbsDataSectionPr
       key="breadcrumbs"
       type="breadcrmbs"
       title={t('Breadcrumbs')}
-      actions={actions}
       data-test-id="breadcrumbs-data-section"
+      actions={actions}
     >
       <ErrorBoundary mini message={t('There was an error loading the event breadcrumbs')}>
-        {searchedCrumbs.length ? (
-          <ScrollBox>
-            <BreadcrumbsTimeline
-              breadcrumbs={searchedCrumbs}
-              virtualCrumbIndex={virtualCrumbIndex}
-              meta={meta}
-              sort={sort}
-              timeDisplay={timeDisplay}
-            />
-          </ScrollBox>
+        {allCrumbs.length ? (
+          <BreadcrumbsTimeline
+            breadcrumbs={summaryCrumbs}
```

Comment:
if summaryCrumbs is the thing getting rendered, it should probably be what the useMemo is returning

Reply:
I'll account for this but I still need `allCrumbs` for the drawer content. Thanks!